scss/cart/cart.css

```diff
@@ -0,0 +1,34 @@
+body {
+  color: #929292;
+  font-family: 'Cormorant', serif; }
+
+h1 {
+  font-size: 40px; }
+
+hr {
+  width: 85%; }
+
+.product-item {
+  max-width: 100%; }
+  @media screen and (min-width: 768px) {
+    .product-item {
+      width: 25%; } }
+
+.item-title {
+  color: #c7b198; }
+
+.quantity-box {
+  height: 40px;
+  width: 40px; }
+
+.proceed-btn {
+  background-color: #c7b198;
+  color: white; }
+
+.cont-shopping {
+  text-decoration: none;
+  color: #929292; }
+  .cont-shopping:hover {
+    color: #929292; }
+
+/*# sourceMappingURL=cart.css.map */
```

Comment:
You got left over code from time you linked the scss file incorrectly, do remove the following files from scss folder:
main.css
main.css.map
cart/cart.css
cart/cart.css.map
contact/contact.css
contact/contact.css.map

Reply:
added these to ignore file

.vscode/*
.DS_Store
debug.log
css/*
scss/*.css
scss/*.css.map